scenes/game/local_game.gd

```diff
@@ -3359,7 +3549,8 @@ func do_boost(performing_player : Player, card_id : int, payment_card_ids : Arra
 	event_queue += events
 	return true
 
-func do_strike(performing_player : Player, card_id : int, wild_strike: bool, ex_card_id : int) -> bool:
+func do_strike(performing_player : Player, card_id : int, wild_strike: bool, ex_card_id : int,
+		opponent_sets_first : bool = false) -> bool:
```

Comment:
as mentioned, this parameter will need to be added to remotegame and the ai versions of this function.